tests/cargo-kani/asm/global_error/doesnt_call_crate_with_global_asm.expected

```diff
@@ -1,3 +1 @@
 error: Crate crate_with_global_asm contains global ASM, which is not supported by Kani. Rerun with `--enable-unstable --ignore-global-asm` to suppress this error (**Verification results may be impacted**).
-error: could not compile `crate_with_global_asm` due to previous error
```

Comment:
This is a useful line that points out the name of the crate that had an error. Is the crate name available elsewhere?

Reply:
There seems to be a bug on Cargo that it counts an extra error if you use `--message-format=json`. Basically, the compiler prints an error message saying how many errors were found. In regular mode, cargo excludes this message, but when emitting `json`, cargo skips that, and counts it as an extra error.

I.e., for this test to pass, I have three options:

1. Remove the check from here.
2. Update the error count:
```
error: could not compile `crate_with_global_asm` due to 2 previous errors
```
3. Update the expected message to not include the number of errors:
```
error: could not compile `crate_with_global_asm`
```

I picked the first since it was the simplest, but I can change. Any preference?

Reply:
I see. The option you went with is fine. I just wanted to make sure the message still gets printed.

Reply:
FYI, I updated this one expected test to have the message that includes the name of the crate and the counter as a sanity check, but I didn't bother adding to all the other tests, because one would hope that Cargo will update that counter eventually. In that case, we would only need to update one test.